sample_config/multiharp160.json

```diff
@@ -25,5 +25,26 @@
             "edge_trigger": "Falling",
             "offset": 0
         }
-    ]
+    ],
+    "row_filter": 
```

Comment:
The `row_filter` configuration JSON here doesn't match the configuration structure defined in `device.rs`. I don't think that the sample config was tested. Can you send me the configurations that you did test with @yuki9638 ?